### DISCUSSION TITLE
Special character replacement in titles

### DISCUSSION THREAD

**Discussion by tunggian** [coddingtonbear/obsidian-web#63] [Q&A] [answered]
There are some web page titles that have special characters in them, but the obsidian title does not support these special characters, for example I

Hopefully the plugin could replace these special characters before saving them in the obsidian

Thank you

**Comment by coddingtonbear** [comment, marked as answer]
This exists -- see the default templates for the `filename` helper. This helper will strip your filename down to a cross-platform compatible set of filesystem-safe characters.

**Reply by tunggian** [reply]
Thank you for your answer, I'm sorry I didn't see it

**Reply by coddingtonbear** [reply]
No worries at all!
